models/amazon_selling_partner__order_items.sql

```diff
@@ -0,0 +1,116 @@
+{{ config(enabled=var('amazon_selling_partner__using_orders_module', true)) }}
+
+with orders as (
+
+    select *
+    from {{ ref('stg_amazon_selling_partner__orders') }}
+),
+
+order_item as (
+
+    select *
+    from {{ ref('stg_amazon_selling_partner__order_item') }}
+),
+
+{% if var('amazon_selling_partner__using_catalog_module', true) %}
+{# Open Q: Should we bring item in here?  #}
```

Comment:
I think it could be a good addition if the user does have the catalog module. I say keep what you have as this is a good and natural extension.

You can remove this comment.

Reply:
removed